### PR TITLE
Milestone two

### PR DESCRIPTION
**Pull request by helciodev**
The features of this milestone are:

1. Project uses create-react-app
2. Unnecessary files are deleted.
3. Project is deployed on Heroku successfully.
4. Project shows buttons for the calculator and respective values.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <div>
+      <Display result="0" />
+      <ButtonPanel />
+    </div>
```
Comment (taiwo2) [inline src/components/App.js]: This code should be wrapped with React.Fragment short syntax

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <div>
+      <Display result="0" />
```
Comment (taiwo2) [inline src/components/App.js]: Default value props  should be set  in display component

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 
@helciodev  well done on this milestone, I commend your effort so far, however, changes are required to get your project approved 
### highlights

- Git flow 👍🏻 
- professional readme file 👍🏻 
- deployment To Heroku 👍🏻 

 ### Improve

### `App` component
- [ ] has children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))
###  `Display` component
- [ ] its result prop should be a String with default value = 0 ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))
- kindly check my Inline comments

### once specific changes are made kindly request another review. thanks

**Review by voscarmv — APPROVED** [review]
# Project status: Approved 💯 ✨ 😎 👍 
### You may merge this branch and move forward ⛙🛣️ 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**